reo/src/data_manager.py

```diff
@@ -1193,18 +1193,21 @@ def finalize(self):
             heating_load = self.heating_load.load_list
         else:
             heating_load = [0.0 for _ in self.load.load_list]
-        if self.cooling_load != None:
+        if self.LoadProfile["annual_cooling_kwh"] > 0.0:
             cooling_load = self.cooling_load.load_list
             # Zero out cooling load for outage hours
             if time_steps_without_grid not in [None, []]:
                 for outage_time_step in time_steps_without_grid:
                     cooling_load[outage_time_step-1] = 0.0
+                self.LoadProfile["year_one_chiller_electric_load_series_kw"] = cooling_load
+                self.LoadProfile["annual_cooling_kwh"] = sum(cooling_load) / self.elecchl_cop * self.steplength
```

Comment:
might want to check this one 1/1*4 = 4, but 1/(1*4)=0.25

Reply:
self.steplength = 8760.0 / self.n_timesteps, so =0.25 if time_steps_per_hour is 4. So multiplying by 0.25 is appropriate for  time_steps_per_hour = 4. Right?

However, I just realized in line 1202 that I'm equating the electric cooling load with `cooling_load` which is thermal, so I need to divide by self.elecchl_cop there. I will push that fix.

Reply:
My bad, I was mixing up self.steplength with time_steps_per_hour. That sounds good!